GiEnJul/ClientApp/src/components/verifyConnection/verifyConnection.tsx

```diff
@@ -11,33 +13,52 @@ interface RouteParameters {
 
 interface VerifyConnection extends RouteComponentProps<RouteParameters> {}
 
+const useStyles = makeStyles({
+  midScreen: {
+    textAlign: "center",
+    top: "50%",
+    left: "50%",
+  },
+});
+
 const VerifyConnection: React.FC<VerifyConnection> = () => {
   const { giverRowKey, recipientRowKey, partitionKey } =
     useParams<RouteParameters>();
+  const classes = useStyles();
+
+  const [success, setSuccess] = useState<boolean | undefined>(undefined);
 
-  const [error, setError] = useState(false);
   const apiservice = new ApiService();
 
   useEffect(() => {
     apiservice
       .post(`verify/${giverRowKey}/${recipientRowKey}/${partitionKey}`, {})
-      .then((response) => setError(!(response.status == 200)));
+      .then((response) => {
+        setSuccess(response.status == 200);
+      });
   }, []);
 
-  if (error)
+  if (success)
+    return (
+      <>
+        <h1 className={classes.midScreen}>Takk til deg!</h1>
```

Comment:
I think material-ui typography should be used here: https://material-ui.com/api/typography/. I think there is a prop that can be added here (e.g. `align="center"`) instead of the midScreen class.